service/worker/batcher/activities.go

```diff
@@ -212,6 +216,35 @@ func (a *activities) getActivityLogger(ctx context.Context) log.Logger {
 	)
 }
 
+func containsStatusCondition(whereClause string) bool {
```

Comment:
We shouldn't care about this as well. We don't want user to start batch terminate job with query like "ExecutionStatus = Completed"

Reply:
The reason I wanted that filter is to prevent user's mistake causing server to pay unnecessary cost.